### PR TITLE
Website: Move EIP badges below description

### PR DESCRIPTION
**Pull request by Pandapip1**
This way, the title/description are a little more obvious.

### CODE REVIEW

File: _layouts/eip.html [hunk 1]
```diff
@@ -18,14 +18,33 @@
 </svg>
 
 <div class="home">
+  <h1 class="page-heading">
+    {% if page.category == "ERC" %}
+      ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    {% elsif page.category != "ERC" %}
+      EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    {% endif %}
```
Comment (SamWilsn) [inline _layouts/eip.html:26]: Might as well clean this up while we're here:

```suggestion
    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}
    -{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
```

Probably should double check that the `-%}` syntax works. I only know it from jinja2 and django.

Comment (Pandapip1) [reply _layouts/eip.html:26]: Why is that necessary? Wouldn't this work just as well, then?
```suggestion
    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
```

File: _layouts/eip.html [hunk 2]
```diff
@@ -18,14 +18,29 @@
...
+
   <span class="h5">
     {% if page.status == "Stagnant" %}
       <span class="badge text-light bg-danger" data-bs-toggle="tooltip" data-bs-title="This EIP had no activity for at least 6 months. This EIP should not be used.">🚧 Stagnant</span>
     {% endif %}
     {% if page.status == "Withdrawn" %}
       <span class="badge text-light bg-danger" data-bs-toggle="tooltip" data-bs-title="This EIP has been withdrawn, and should not be used.">🛑 Withdrawn</span>
     {% endif %}
-    {% if page.status == "Draft" or	page.status == "Review" %}
+    {% if page.status == "Draft" or page.status == "Review" %}
```
Comment (Pandapip1) [inline _layouts/eip.html:43]: I'm not sure why this changed and I'm a little scared.

Comment (Pandapip1) [reply _layouts/eip.html:43]:
```suggestion
    {% if page.status == "Draft" or	page.status == "Review" %}
```

Comment (Pandapip1) [reply _layouts/eip.html:43]: Oh. It was previously a tab. I don't know how that happened.

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: _layouts/eip.html [hunk 3]
```diff
@@ -18,14 +18,29 @@
 </svg>
 
 <div class="home">
+  <h1 class="page-heading">
+    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    <a href="{{ page["discussions-to"] | uri_escape }}" class="no-underline">
```
Comment (SamWilsn) [inline _layouts/eip.html:23]: Hadn't we switched this to:

```suggestion
    <a href="{{ page.discussions-to | uri_escape }}" class="no-underline">
```

**Comment by Pandapip1** [comment]
Superseded by #6518

**Pull request closed without merging**